### DISCUSSION TITLE
Use wire:poll on regular component.

### DISCUSSION THREAD

**Discussion by rickdubiel** [livewire/livewire#4483] [1. Help] [answered]
I'm trying out the simple polling example here: https://laravel-livewire.com/docs/2.x/polling

All works fine at that point, but if I try to use `wire:poll` on a component I'm getting the following error:

> Livewire\Exceptions\MethodNotFoundException
> Unable to call component method. Public method [wire:poll] not found on component: [poll-test] 

Here's my regular component (resources/views/components/block.blade.php):
```
<div {{ $attributes }}>
    {{ $slot }}
</div>
```

And here's the livewire component (resources/views/livewire/poll-test.blade.php):
```
<x-block wire:poll>
    Current time: {{ now() }}
</x-block>
```

Does this just not work when passing as attributes into regular components like this?

**Comment by joshhanley** [comment, marked as answer]
@rickdubiel that's actually an issue with how blade components render, for some reason, if you pass an attribute in without a value, it renders the value as the attribute like `wire:poll="wire:poll"`. If you check out your rendered html in dev tools you should see something like that.

I think the fix for that is to just pass an empty string, so like `<x-block wire:poll=''>`.

Hope this helps!
